pom.xml

```diff
@@ -1257,6 +1257,9 @@
     </profile>
     <profile>
       <id>ci</id>
+      <modules>
+        <module>aggregates/jetty-all</module>
+      </modules>
```

Comment:
@olamy so this module was only built on CI, not during the release.
Why did you have problems during the release then?

Reply:
nope the only way. It was only build during release and not on ci :)
so now it will be check on ci

Reply:
failure validated here :) https://jenkins.webtide.net/job/jetty.project/job/jetty-10.0.x_fix_dependency/2/

Reply:
Doh!